tests/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj

```diff
@@ -14,6 +14,7 @@
     <ProjectReference Include="..\..\..\src\Components\Aspire.MongoDB.Driver\Aspire.MongoDB.Driver.csproj" />
     <ProjectReference Include="..\..\..\src\Components\Aspire.MySqlConnector\Aspire.MySqlConnector.csproj" />
     <ProjectReference Include="..\..\..\src\Components\Aspire.Npgsql\Aspire.Npgsql.csproj" />
+    <ProjectReference Include="..\..\..\src\Components\Aspire.Oracle.EntityFrameworkCore.Database\Aspire.Oracle.EntityFrameworkCore.Database.csproj" />
```

Comment:
Once https://github.com/dotnet/aspire/pull/1217 is merged, we should update this test to follow the new pattern established there.

Reply:
Ok, I will get the updates and update the tests

Reply:
Done